sdk/resourcemanager/Azure.ResourceManager/src/Resources/Custom/TenantResource.cs

```diff
@@ -16,8 +16,10 @@ namespace Azure.ResourceManager.Resources
     [CodeGenSuppress("GetAsync", typeof(CancellationToken))]
     [CodeGenSuppress("GetAvailableLocations", typeof(CancellationToken))]
     [CodeGenSuppress("GetAvailableLocationsAsync", typeof(CancellationToken))]
+#pragma warning disable CA1825 // Avoid zero-length array allocations
```

Comment:
@m-nash this was necessary with Visual Studio 17.6.0 Preview 2.0 and has been for a little while, IIRC. We may need a constructor overload because you can't pass `Array.Empty<Type>()` since it's not a constant expression. Or is this a known issue?

Reply:
I've seen this issue locally - is it possible to fix the issue rather than disable the warning?

Reply:
Would it make sense to file an issue to track the work?